### PR TITLE
Clickable link to OS based installation for Sympy

### PR DESCRIPTION
**Pull request by 75072107**
#### Brief description of what is fixed or changed
Added a Clickable link labelled "HERE" for OS based installation of Sympy

#### Other comments
This is my First PR. Advice and criticism equally appreciated.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by asmeurer** [comment]
The style guide README intentionally points to the file instead of the url.

**Comment by 75072107** [comment]
Oh.. sorry then.My bad. This was my first PR. First time contributing to a real project, I mean. Thanks for the help.

**Comment by oscarbenjamin** [comment]
The change to README.rst looks good but I think that the other change is not needed. There are ways to undo a change in a pull request e.g. by editing the files, committing  the change and pushing.

In future I recommend that you use a branch for making pull requests rather than committing on your master branch.

File: README.rst [hunk 1]
```diff
@@ -38,6 +38,8 @@ to ask us anything there. We have a very welcoming and helpful community.
 Download
 --------
 
+Installation methods for Windows, MacOS and Linux are available here :- "https://docs.sympy.org/latest/documentation-style-guide.html#installation"
+
```
Comment (moorepants) [inline README.rst]: Isn't this the instructions for installing a development environment for building the documentation? I don't think that is what we want to point end users to.

**Comment by czgdp1807** [comment]
IMHO, this can be closed, as most people install through `conda` or `pip`. Though if you feel that this should re-opened and continued then please re-open this PR or if you are unable to do so, just let us know in the comments. Thanks for your contributions.

**Pull request closed without merging**